test/support/hipchat_client.ex

```diff
@@ -20,8 +20,8 @@ defmodule Cog.Test.Support.HipChatClient do
 
   def start_link() do
     api_token = get_env!("HIPCHAT_USER_API_TOKEN")
-    xmpp_jid = get_env!("HIPCHAT_USER_JID")
-    xmpp_password = get_env!("HIPCHAT_USER_PASSWORD")
+    xmpp_jid = get_env!("HIPCHAT_USER_JABBER_ID")
```

Comment:
Is `HIPCHAT_USER_JID` something we should deprecate, or was that never really exposed to the public?